kerastuner/distribute/oracle_servicer.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2019 The Keras Tuner Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from concurrent import futures
+import grpc
+import os
+import time
+
+from ..engine import hyperparameters as hp_module
+from ..protos import service_pb2
+from ..protos import service_pb2_grpc
+
+
+class OracleServicer(service_pb2_grpc.OracleServicer):
```

Comment:
What was the motivation for the name "OracleServicer"? What would be some other options?

Reply:
Done, went with oracle_chief